### ISSUE TITLE
Serving some assets from /pages seems broken

### ISSUE THREAD

**Issue by strzibny** [sitepress/sitepress#37]
Now that I know I can put anything to /pages to be publicaly available I realized that while this works on *compile* step, it doesn't work corrently when developing the site (when these files are served by Sitepress).

Serving font files seems to be broken:

```
127.0.0.1 - - [08/Aug/2023:16:46:59 CEST] "GET /assets/fa-brands-400.woff2 HTTP/1.1" 404 0
http://localhost:8080/webfonts/fa-brands-400.ttf -> /assets/fa-brands-400.woff2
Started GET "/favicon.ico" for 127.0.0.1 at 2023-08-08 16:47:00 +0200
Processing by SiteController#show as HTML
  Parameters: {"resource_path"=>"favicon.ico"}
Sitepress resolved asset pages/favicon.ico
Sent file pages/favicon.ico (0.2ms)
Completed 200 OK in 7ms (Allocations: 2217)
```

Putting these font files in `/pages` is especially convenient since the reference to these paths are in CSS files generated by font-awesome.

**Comment by bradgessler** [comment]
I think I’m going to remove Sprockets from Sitepress. I threw it in there because “asset pipeline”, but as we’ve learned over and over again, they’re always a huge pain and seem to require superhuman levels of mysticism to configure.

Throwing the files into `./pages` is the direction I want to go with Sitepress since it’s just so easy and intuitive.

Are you running Sitepress stand-alone or embedded in Rails?

**Comment by bradgessler** [comment]
I just removed Sprockets from https://sitepress.cc to see how well it would work and, news flash; not having an asset pipeline is awesome.

the tl;dr is:

1. Delete the `./assets/config/manifest.js` file
2. Move all of the files and folders from `./assets/` to `./pages`. For example the `./assets/images` folder should be moved to `./pages/images`.
3. That's it. The image helpers, etc. should pick up the pages from those directories.

Here's the commit I in Sitepress where I removed the asset pipeline: https://github.com/sitepress/website/commit/da64c33ccba798834dd6ae2c2e6427a04843af0a
